content/docs/deployment/csmoperator/_index.md

```diff
@@ -38,6 +38,7 @@ Dell CSM Operator can be installed manually or via Operator Hub.
 #### Operator Installation on a cluster without OLM
 
 1. Clone the [Dell CSM Operator repository](https://github.com/dell/csm-operator).
+2. (Optional) If using a local Docker image, edit the `deploy/operator.yaml` file and set the image name for the CSM Operator Deployment.
 2. `cd csm-operator`
```

Comment:
unless we cd to the operator , we cannot run the command below in point 3 @tdawe , can we add it back

Reply:
Ok re-ordering these steps